x-pack/plugins/ml/public/application/explorer/explorer_charts/explorer_anomalies_container.tsx

```diff
@@ -23,6 +23,7 @@ import type { AnomaliesTableData } from '../explorer_utils';
 
 interface ExplorerAnomaliesContainerProps {
   id: string;
+  mode?: string;
```

Comment:
Can this have union of explicit values?

Reply:
Also, it looks like this is always supplied, so doesn't need to be optional

Reply:
Yep, higher up in the component chain, it will either not be provided or be "embeddable" - but by this point it will always be provided. Updated in 41c37ae127f9a44b4f83250e2b3fe1def6953a11